src/Toml/Codec/Di.hs

```diff
@@ -136,7 +136,7 @@ diwrap
        (Coercible a b)
     => TomlCodec a
     -> TomlCodec b
-diwrap = dimap coerce coerce
+diwrap = coerce
```

Comment:
Current mood: reimplementing `mtl` just to use `coerce` :rofl:

Reply:
:bad_guys: :rofl: